Include/object.h

```diff
@@ -567,7 +569,7 @@ PyAPI_FUNC(int) PyObject_CallFinalizerFromDealloc(PyObject *);
 /* Same as PyObject_Generic{Get,Set}Attr, but passing the attributes
    dict as the last parameter. */
 PyAPI_FUNC(PyObject *)
-_PyObject_GenericGetAttrWithDict(PyObject *, PyObject *, PyObject *);
+_PyObject_GenericGetAttrWithDict(PyObject *, PyObject *, PyObject *, int);
```

Comment:
Would this change be backward incompatible? Or is this function is not covered by the backwards compatibility guarantees?

Reply:
AFAIK, APIs starting with underscore are private.
Third party libs shouldn't expect backward compatibility.
I don't know should I add `Include/internal/object.h` or not.

Reply:
OK, I see. But I don't know about ``Include/internal/object.h`` either. Probably @serhiy-storchaka can advise on this.

Reply:
Changing the signature of `_PyObject_GenericGetAttrWithDict()` LGTM. As for `Include/internal/object.h`, left this to other issue.